### ISSUE TITLE
Warping into damgerous hostilities

### ISSUE THREAD

**Issue by zelbov** [RavienGaming/Reforged-Eden-2#29]
### Reforged Eden 2 Build Number

1

### Game mode

Single Player

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

Still there is a possibility of warping straight into close proximity with hostile POIs that might be considered too difficult on certain game stages. Just warped into sector and appeared in like 800m away from Naval Shipyard and got some loud and scary hits with shield penetrating missiles.

Couldnt manage to save sector name and seed for a report though, sry

**Comment by RavienGaming** [comment]
Yep this is a game issue. The developers would need to add some kind of protection or safety mechanic to prevent players from being attacked after warping or from warping in on top of enemy POIs.

**Comment by RavienGaming** [comment]
Relevant suggestion thread on the official Empyrion Discord to add a short immunity from being targeted by NPC turrets after warping:
https://discord.com/channels/556235494640582666/1234994817495203882
 _(thread may have been deleted for those looking for this in the future)_
